lib/components/viewers/nearby/styled.tsx

```diff
@@ -20,15 +20,13 @@ export const NearbySidebarContainer = styled.ol`
   gap: 1em;
   padding: 0 1em;
   list-style: none;
+  overflow-y: scroll;
+  height: 90%;
```

Comment:
This feels like a bit of a hack to deal with the location field. Could we not instead just wrap the location field in an `li`? Or move the `ol` styles to a div and then have an `ol` that has no styles?